Java/Java.sublime-syntax

```diff
@@ -159,7 +159,7 @@ contexts:
   constant-expressions:
     - include: groups
     - include: literals
-    - include: punctuations
+    - include: punctuation
```

Comment:
Please do not mess with trailing `s` in context names.

Be its spelling incorrect or not, it describes whether a context pops or not. Changing this may cause bugs in various languages.